services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/component/BoosterNotificationStructureProvider.java

```diff
@@ -25,31 +19,28 @@
 @Component
 public class BoosterNotificationStructureProvider {
 
-  private static final Logger logger = LoggerFactory.getLogger(BoosterNotificationStructureProvider.class);
-
   public static final String EXPORT_BINARY_FILENAME = "export.bin";
```

Comment:
see comment frmo @ioangut ---> we could extract this as a application property

Reply:
I think we have one already, it is under tek-export: file-name. But I am not sure we should use that one because we can create confusion then.

Reply:
@ioangut well.... Then we have already confusion. If the existing one is for TEK-Exports then either we take a global archive name or we make distinct archive names ( which have then all the same value^^).
@alexmoroianu What do you think?

Reply:
Hi. I created a default-archive-name in distribution. And also use specific ones for each service. is this fine ?